fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+                  			"title": "Basic White Tee",
+                  			"price": 19.99,
+                  			"quantity": 2
+            			}
+              		]
+            	}
+            }'
+            
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              ShopifyAPI::DraftOrder.create({
+                email: "paul.norman@example.com",
+                customer: {
+                    email: "paul.norman@example.com",
+                    first_name: "Paul",
+                    last_name: "Norman"
+                },
+                shipping_address: {
+                    first_name: "Jane",
+                    last_name: "Smith",
+                    address1: "123 Fake Street",
+                    phone: "777-777-7777",
+                    city: "Fakecity",
+                    province: "Ontario",
+                    country: "Canada",
+                    zip: "K2P 1L4"
+                },
+                line_items: [{
+                    custom: true,
+                    title: "Basic White Tee",
+                    price: 19.99,
+                    quantity: 2
+                }]
+              })    
+   
+            
+  '/admin/draft_orders/#{id}.json':
+    get:
+      description: Receive a single draft order
+      tags:
+        - Get
+      parameters:
+        - $ref: '#/parameters/fields'
+        - name: id
+          in: path
+          description: draft order `id`.
+          required: true
+          type: integer
+          format: int64
+      responses:
+        '200':
+          description: A single draft order
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'    
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X GET -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/57760134.json
+      
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              draft_order = ShopifyAPI::DraftOrder.find(#{id})
+            
+ 
+      
+    put:
+      description: Update a draft order
+      tags:
+        - Update
+      parameters:
+      
+        - name: id
+          in: path
+          description: draft order `id`.
+          required: true
+          type: integer
+          format: int64
+        - name: draft_order
+          in: body
+          description: draft_order to create.
```

Comment:
I don't have the context here, but this line looks off to me on the page:

![screen shot 2016-11-28 at 1 55 04 pm](https://cloud.githubusercontent.com/assets/10617892/20681597/4e7c8f9a-b572-11e6-997b-9a279beadc0c.png)

Should draft_order be in code formatting? Would it make sense to put this line in a sentence form, or should we just remove the period at the end and roll with it?